cves/CVE-2015-0220.yml

```diff
@@ -196,8 +236,10 @@ i18n:
 
     Answer should be boolean. Write a note about how you came to the conclusions
     you did.
-  answer:
-  note:
+  answer: false
+  note: |
+    As this was whitespace in a url, it is a universal language property by
```

Comment:
I'm honestly confused if you are referring to a universal language property of Python, or a universal language property that all languages have spaces, which they do not take for example the Japanese and Chinese languages respectively.